lib/wallaroo/core/source/tcp_source/framed_source_notify.pony

```diff
@@ -158,13 +163,43 @@ class TCPFramedSourceNotify[In: Any val] is TCPSourceNotify
       end
     end
 
-  fun ref update_router(router: Router) =>
-    _router = router
+  fun ref _report_message(is_finished: Bool, last_ts: U64,
```

Comment:
`_report_metrics`?

Reply:
turns out this wasn't being used, so removed.